environments/key4hep-common/packages.yaml

```
@@ -140,6 +140,12 @@ packages:
     target: []
     providers: {}
     compiler: []
+  matplotlib:
+    variants: backend=qt5agg
+    version: []
+    target: []
+    providers: {}
+    compiler: []
```

```suggestion
  py-matplotlib:
    variants: backend=qt5agg
```

The name is `py-matplotlib`